engine/swarm/key-concepts.md

```diff
@@ -14,30 +14,33 @@ are built using **SwarmKit**. Engines participating in a cluster are
 running in **swarm mode**. You enable swarm mode for the Engine by either
 initializing a swarm or joining an existing swarm.
 
-A **swarm** is a cluster of Docker Engines where you deploy
-[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI includes the commands for
-swarm management, such as adding and removing nodes. The CLI also includes the
-commands you need to deploy services to the swarm and manage service
-orchestration.
+A **swarm** is a cluster of Docker engines, or _nodes_, where you deploy
+[services](key-concepts.md#Services-and-tasks). The Docker Engine CLI includes
+the commands for swarm management, such as adding and removing nodes. The CLI
+also includes the commands you need to deploy services to the swarm and manage
+service orchestration.
 
-When you run Docker Engine outside of swarm mode, you execute container
-commands. When you run the Engine in swarm mode, you orchestrate services.
+When you run Docker outside of swarm mode, you execute container
+commands. When you run the Docker in swarm mode, you orchestrate services.
 
 ## Node
 
-A **node** is an instance of the Docker Engine participating in the swarm.
+A **node** is an instance of the Docker engine participating in the swarm.
 
 To deploy your application to a swarm, you submit a service definition to a
 **manager node**. The manager node dispatches units of work called
 [tasks](key-concepts.md#Services-and-tasks) to worker nodes.
 
 Manager nodes also perform the orchestration and cluster management functions
-required to maintain the desired state of the swarm. Manager nodes elect a single leader to conduct orchestration tasks.
+required to maintain the desired state of the swarm. Manager nodes elect a
+single leader to conduct orchestration tasks.
 
-**Worker nodes** receive and execute tasks dispatched from manager nodes. By
-default manager nodes are also worker nodes, but you can configure managers to
-be manager-only nodes. The agent notifies the manager node of the current
-state of its assigned tasks so the manager can maintain the desired state.
+**Worker nodes** receive and execute tasks dispatched from manager nodes.
+By default manager nodes are also worker nodes, but you can configure managers
```

Comment:
Maybe something like: By default, manager nodes can also run services, but you can configure them to only perform management tasks.